build.sh

```diff
@@ -441,15 +433,6 @@ case "$COMMAND" in
         kill_object_server
         # Add a short delay, so file system doesn't complain about files in use
         sleep 1
-        package="${source_root}/sync"
-        for file in "$package"/realm-object-server-*; do
-            if [ -d "$file" ]; then
-                package="$file"
-                break
-            fi
-        done
-        rm -rf "$package/object-server/root_dir/"
-        rm -rf "$package/object-server/temp_dir/"
```

Comment:
Don't we still need to nuke the server's on-disk state?

Reply:
I think doing so would be redundant, since we call this method before any tests are run but immediately after the latest object server is downloaded and installed, which involves deleting any existing server state.

Reply:
Isn't it invoked by the first object server test case that runs? If you run the tests twice in a row you'd expect this to clear the state between the two runs, but it no longer does. The download / installation isn't repeated the second time you run the tests.

Reply:
When you say twice in a row, do you mean running `sh build.sh verify-osx-object-server`, waiting until it completes, then running it again? If so, `download-object-server` will be invoked by the first test that runs each time.

That being said, I think we should clear the state anyways, since it's surprising that a method called `reset-object-server` doesn't clear the state.

Reply:
I'm talking about running tests directly within Xcode.

Reply:
Oh, okay. In that case I should probably add some affordances to running the tests without using build.sh, because it's never worked quite right (e.g. you have to manually download the ROS).

Reply:
Having to run `sh build.sh download-object-server` once before running tests in Xcode isn't that much of a burden. There are more important things to worry about :-)